Description: Fix the wired error by GCC7 that fails to match the correct parent method.
Author: Kai-Chung Yan
Forwarded: not-needed
--- a/libutils/include/utils/Vector.h
+++ b/libutils/include/utils/Vector.h
@@ -256,7 +256,7 @@
 
 template<class TYPE> inline
 const Vector<TYPE>& Vector<TYPE>::operator = (const Vector<TYPE>& rhs) const {
-    VectorImpl::operator = (static_cast<const VectorImpl&>(rhs));
+    VectorImpl::operator = (rhs);
     return *this;
 }
 
